geocity/apps/submissions/steps.py

```diff
@@ -83,6 +83,13 @@ def get_selectable_categories(submission=None, category_tags=None):
     ).distinct()
 
 
+def get_selectable_form(user, quick_access_slug):
+    forms = models.Form.objects.filter(quick_access_slug=quick_access_slug)
+    if not user.has_perm("submissions.view_private_form"):
+        forms = forms.filter(is_public=True)
```

Comment:
@AlexandreJunod I think there is one filter to add here, do you confirm ?

Reply:
What do you mean by one filter ? If user cannot view private forms, we should show only the ones in public that correspond to the quick_access_log.